server/sonar-server/src/main/java/org/sonar/server/setting/ws/ResetAction.java

```diff
@@ -140,7 +140,7 @@ private static ResetRequest toWsRequest(Request request) {
 
   private void checkPermissions(Optional<ComponentDto> component) {
     if (component.isPresent()) {
-      userSession.checkComponentUuidPermission(UserRole.ADMIN, component.get().uuid());
+      userSession.checkComponentPermission(UserRole.ADMIN, component.get());
     } else {
       userSession.checkPermission(GlobalPermissions.SYSTEM_ADMIN);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "checkPermission"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)